api/dependencies/file.py

```
@@ -90,158 +126,148 @@ def detect_mime_type_fallback(file_content: bytes, filename: str='') ->str:
-        threats_detected.append(
-            f'MIME type mismatch: declared={content_type}, detected={detected_type}'
-            )
+        threats_detected.append(f"MIME type mismatch: declared={content_type}, detected={detected_type}")
         security_score += 0.4
     _, ext = os.path.splitext(filename.lower())
     if ext in DANGEROUS_EXTENSIONS:
-        threats_detected.append(f'Dangerous file extension: {ext}')
+        threats_detected.append(f"Dangerous file extension: {ext}")
         security_score += 0.8
     for pattern in MALWARE_BYTE_PATTERNS:
         if pattern.encode() in file_content:
```

Pylint (3.3.8)

[error] Instance of 'bytes' has no 'encode' member

(E1101)